libraries/index.html

```diff
@@ -0,0 +1,121 @@
+      download the 4.x bundle. The .mpy files are designed to take up less space on
+      your CIRCUITPY drive. Unless you have a specific reason, choose the appropiate
+      .mpy library bundle!
+    </p>
+    <p>
+      <b>To install</b>, download the appropriate bundle for your version of CircuitPython.
+      Unzip the file, open the resulting folder and find the lib folder. Open the lib
+      folder and find the library files you need to load. Create a lib folder on your
+      CIRCUITPY drive. Copy the individual library files you need to the lib folder
+      on your CIRCUITPY drive.
+    </p>
+    <p>
+      You can always find the
+      <a href="https://github.com/adafruit/Adafruit_CircuitPython_Bundle/releases/latest">latest releases of the libraries bundle</a>
+      on GitHub.
+    <div class="release-section">
+      <div id="adafruit-circuitpython-bundle-4.x-mpy">
+        <h3>Bundle Version 4.x</h3>
+        <p>
+          This bundle is built for use with CircuitPython 4.x.x. If you are using
+          CircuitPython 4, please download this bundle.
+        </p>
+      </div>
+      <div id="adafruit-circuitpython-bundle-3.x-mpy">
+        <h3>Bundle Version 3.x</h3>
+        <p>
+          This bundle is built for use with CircuitPython 3.x.x. If you are using
+          CircuitPython 3, please download this bundle.
+        </p>
+      </div>
+      <div id="adafruit-circuitpython-bundle-py">
+        <h3>Bundle Version py</h3>
+        <p>
+          This bundle is the uncompiled source code for every library. It is not
+          intended for general use! It is only recommended if you need to edit
+          a library file. This bundle works with all versions of CircuitPython.
+        </p>
+      </div>
+      <div id="adafruit-circuitpython-bundle-examples">
+        <h3>Bundle Examples</h3>
+        <p>
+          Every CircuitPython library has examples to go along with it. This bundle
+          contains every library example. This bundle will have examples that
+          require the latest version of CircuitPython, and therefore contains files
+          that may not work with all versions.
+        </p>
+      </div>
+    </div>
+    <h2>Contributing</h2>
+    <p>
+      If you'd like to contribute to the CircuitPython project, the CircuitPython
+      libraries are a great way to begin. Everyone is welcome to contribute!
+      Regardless of your experience level, we have many resources to get you
+      started. Check out the Contributing page for detailed information!
+    </p>
+    <p>
+      Find out more about how you can
+      <a href="{{ "/libraries/contributing" | relative_url  }}" title="Contributing Page">contribute to CircuitPython libraries</a>.
+    </p>
+    <!-- (can we even include the date like this?) -->
+    <h2>Library Activity for Week of  {{ site.data.libraries.updated_at | date: "%a, %b %d, %Y" }}</h2>
+    <p>
+      Thank you to this week's contributors:
```

Comment:
I'd like this to be a standalone line: `Thank you to all of our contributors!`

We're going to add the number of pull requests merged to the json file. Then I would like the following line followed by two bullet points:
```
This week there were [x] pull requests merged:
* Authors: [contributor names]
* Reviewers: [reviewer names]
```
The list of reviewers is also contained in the json file.